spec/index.bs

```diff
@@ -429,6 +435,20 @@ Applications can hold onto multiple [=adapters=] at once (via {{GPUAdapter}})
 and two of these could refer to different instances of the same physical
 configuration (e.g. if the GPU was reset or disconnected and reconnected).
 
+An [=adapter=] has the following internal slots:
+
+<dl dfn-type=attribute dfn-for=adapter>
+    : <dfn>\[[extensions]]</dfn>, of type sequence<{{GPUExtensionName}}>, readonly
+    ::
+        The extensions which can be used to create devices on this adapter.
+
+    : <dfn>\[[limits]]</dfn>, of type {{GPULimits}}, readonly
+    ::
+        The maximum allowed limits which can be used to create devices on this adapter.
+
+        Each adapter limit must be no less than its default value in {{GPULimits}}.
```

Comment:
Some limits are minimums or other things, see for example "Table 30. Required Limits" in [Vulkan's Limits section](https://www.khronos.org/registry/vulkan/specs/1.1/html/chap31.html#limits).